README.md

```diff
@@ -38,6 +38,27 @@ cmake [-G generator] [-DYAML_BUILD_SHARED_LIBS=on|OFF] ..
 
 **Note:** To clean up, just remove the `build` directory.
 
+## How to Integrate it within your project using CMake
+
+You can use for example FetchContent :
+
+```cmake
+include(FetchContent)
+
+FetchContent_Declare(yaml-cpp
+  GIT_REPOSITORY https://github.com/jbeder/yaml-cpp.git
+  GIT_TAG 1b50109f7bea60bd382d8ea7befce3d2bd67da5f)  # The latest yaml-cpp commit at the time of writing.
```

Comment:
Rather than embed a specific example version, could you just put a placeholder e.g. <tag_name> or something so that it forces people to check the latest version?